[V2] svg

YeSeulBae opened the pull request.

YeSeulBae committed:
Add files via upload

YeSeulBae committed:
Delete [v2] SVG 사용.md

파일 변경으로 삭제 후 재업로드 하겠습니다

YeSeulBae committed:
Add files via upload

sujinleeme commented on [v2] SVG 사용.md:
```diff
@@ -0,0 +1,95 @@
+[v2] SVG 사용
```
마크다운 양식을 지켜 제목 추가해주세요.

sujinleeme commented on [v2] SVG 사용.md:
```diff
@@ -0,0 +1,95 @@
+
+이번 장은 SVG 아이콘을 더 멋지고 정확하게 사용할 수 있도록 하는, 이전에 다뤘던 "CSS in React"를 기반으로 합니다. SVG는 다른 스타일링 혹은 스타일링이 전혀 없어도 괜찮습니다. 
+
+이 SVG 아이콘은 [Flaticon's Freepick]에 있습니다. 이 사이트의 많은 SVG는 원작자를 남기면 무료로 사용할 수 있습니다. (https://www.flaticon.com/free-icon/check_109748) 여기서 SVG로 아이콘을 다운로드 할 수 있으며 프로젝트 *src/check.svg*파일에 적용하면 됩니다. 이 파일을 다운로드 받는 것이 좋으나, 더 자세히 알기 위해 SVG 정의를 살펴보면 아래와 같습니다.
```
마크다운이 깨져있어요  
`[Flaticon's Freepick](https://www.flaticon.com/free-icon/check_109748)`

sujinleeme commented on [v2] SVG 사용.md:
```diff
@@ -0,0 +1,95 @@
+
+~~~~~~~
+
+create-react-app을 다시 쓰기 때문에, React components(CSS와 유사한)로 SVG를 바로 import 할 수 있습니다. *src/App.js*에서 SVG를 import 하기 위해 아래 syntax를 사용합니다.
```
import 는 가져올 ~로 수정해주세요. syntax 문법으로 교체해주세요.

sujinleeme commented on [v2] SVG 사용.md:
```diff
@@ -0,0 +1,95 @@
+
+{title="src/App.js",lang="javascript"}
+~~~~~~~
+const Item = ({ item, onRemoveItem }) => (
```
코드를 원본과 동일하게 해주세요. 
들여쓰기가 잘못 되어있어요.

sujinleeme commented on [v2] SVG 사용.md:
```diff
@@ -0,0 +1,95 @@
+}
+~~~~~~~
+
+create-react-app 프로젝트에서 별도의 설치 없이 SVG를 쉽게 사용할 수 있습니다. 하지만 여러분이 Webpack 같은 빌드 도구와 함께 scratch로 React 프로젝트를 만든다면, 여러분이 직접 관리해야 하므로 다릅니다. SVG는 여러분의 애플리케이션을 좀 더 사용하기 쉽게 만들어 주기 때문에 활용할 수 있을 때마다 사용하길 권장합니다.
```
scratch 번역해주세요.
webpack을 웹팩으로 수정해주세요.

sujinleeme commented on [v2] SVG 사용.md:
```diff
@@ -0,0 +1,95 @@
+
+create-react-app 프로젝트에서 별도의 설치 없이 SVG를 쉽게 사용할 수 있습니다. 하지만 여러분이 Webpack 같은 빌드 도구와 함께 scratch로 React 프로젝트를 만든다면, 여러분이 직접 관리해야 하므로 다릅니다. SVG는 여러분의 애플리케이션을 좀 더 사용하기 쉽게 만들어 주기 때문에 활용할 수 있을 때마다 사용하길 권장합니다.
+
+* 실습하기
```
마크다운으로 헤드라인을 수정해주세요.

sujinleeme commented on svg.md:
```diff
@@ -0,0 +1,84 @@
+## SVGs in React
```
이 파일은 삭제해주세요.

sujinleeme commented:
@YeSeulBae 폴더명과 위치를 원본과 동일하게 해야 합니다.  한국어로 되어 있는데 파일 이름을 svg.md 영문으로 고치고 위치도 원본과 동일하게 이동시켜주세요. https://github.com/the-road-to-learn-react/the-road-to-react/tree/master/manuscript/react-styling

YeSeulBae committed:
Delete svg.md

sujinleeme commented:
@YeSeulBae 파일제목을 원본과 동일하게 해서 커밋해주세요.

YeSeulBae committed:
Update and rename [v2] SVG 사용.md to [v2] svg.md

YeSeulBae committed:
Update and rename [v2] svg.md to [v2]svg.md

YeSeulBae committed:
Update [v2]svg.md

YeSeulBae committed:
Rename svg.md to [v2] SVG 사용

YeSeulBae committed:
Update [v2] SVG 사용

YeSeulBae committed:
Update svg.md

sujinleeme commented on manuscript/react-styling/[v2] SVG 사용:
```diff
@@ -0,0 +1,88 @@
+## SVG 사용
```
이 파일 삭제해주세요.

sujinleeme commented on manuscript/react-styling/svg.md:
```diff
@@ -1,10 +1,10 @@
+## SVG 사용
 
-To create a modern React application, we'll likely need to use SVGs. Instead of giving every button element text, for instance, we might want to make it lightweight with an icon. In this section, we'll use a scalable vector graphic (SVG) as an icon in one of our React components.
+최신 React 애플리케이션을 만들기 위해, SVG를 사용하는 것이 좋습니다. 모든 버튼 개체 텍스트를 만드는 대신, 예를 들면, 여러분들은 아이콘으로 좀 더 가볍게 만들 수 있습니다. 이번에는 React components 중 하나인 scalable vector graphic (SVG)를 사용해 보겠습니다. 
```
리액트로 변경
리액트 컴포넌트, 
scalable vector graphic -> 확장 가능한 그래픽으로 변경해주세요.

sujinleeme commented on manuscript/react-styling/svg.md:
```diff
@@ -1,10 +1,10 @@
+최신 React 애플리케이션을 만들기 위해, SVG를 사용하는 것이 좋습니다. 모든 버튼 개체 텍스트를 만드는 대신, 예를 들면, 여러분들은 아이콘으로 좀 더 가볍게 만들 수 있습니다. 이번에는 React components 중 하나인 scalable vector graphic (SVG)를 사용해 보겠습니다. 
 
-This section builds on the "CSS in React" we covered earlier, to give the SVG icon a good look and feel right away. It's acceptable to use a different styling approach, or no styling at all, though the SVG might look off without it.
+이번 장은 SVG 아이콘을 더 멋지고 정확하게 사용할 수 있도록 하는, 이전에 다뤘던 "CSS in React"를 기반으로 합니다. SVG는 다른 스타일링 혹은 스타일링이 전혀 없어도 괜찮습니다. 
```
CSS in React 번역해주세요. 이전 번역 문서 참고해주세요.

sujinleeme commented on manuscript/react-styling/svg.md:
```diff
@@ -1,10 +1,10 @@
+이번 장은 SVG 아이콘을 더 멋지고 정확하게 사용할 수 있도록 하는, 이전에 다뤘던 "CSS in React"를 기반으로 합니다. SVG는 다른 스타일링 혹은 스타일링이 전혀 없어도 괜찮습니다. 
 
-This icon as SVG is taken from [Flaticon's Freepick](https://www.flaticon.com/authors/freepik). Many of the SVGs on this website are free to use, though they require you to mention the author. You can download the icon from [here](https://www.flaticon.com/free-icon/check_109748) as SVG and put it in your project as *src/check.svg*. Downloading the file is the recommended way, however, for the sak ...
+이 SVG 아이콘은 [Flaticon's Freepick](https://www.flaticon.com/authors/freepik).에 있습니다. 이 사이트의 많은 SVG는 원작자를 남기면 무료로 사용할 수 있습니다. (https://www.flaticon.com/free-icon/check_109748) 여기서 SVG로 아이콘을 다운로드 할 수 있으며 프로젝트 *src/check.svg* 파일에 적용하면 됩니다. 이 파일을 다운로드 받는 것이 좋으나, 더 자세히 알기 위해 SVG 정의를 살펴보면 아래와 같습니다.
```
[Flaticon's Freepick](https://www.flaticon.com/authors/freepik). 번역해주세요 
뒤에 . 삭제해주세요.

sujinleeme commented on manuscript/react-styling/svg.md:
```diff
@@ -23,7 +23,7 @@ This icon as SVG is taken from [Flaticon's Freepick](https://www.flaticon.com/au
 ~~~~~~~
 
-Because we're using create-react-app again , we can  import SVGs (similar to CSS) as React components right away. In *src/App.js*, use the following syntax for importing the SVG:
+create-react-app을 다시 쓰기 때문에, React components(CSS와 유사한)로 SVG를 바로 가져올 수 있습니다. *src/App.js*에서 SVG를 가져오기 위해 아래 문법을 사용합니다.
```
React components 번역해주세요.

sujinleeme commented on manuscript/react-styling/svg.md:
```diff
@@ -36,7 +36,7 @@ import { ReactComponent as Check } from './check.svg';
 ~~~~~~~
 
-We are importing an SVG, and this works for many different uses for SVGs (e.g. logo, background). Instead of a button text, pass the SVG component as a `height` and `width` attribute:
+SVG를 가져오게 되면, (e.g. logo, background) 다양한 SVG를 활용할 수 있습니다. 버튼 텍스트 대신, SVG component로 'height'와 'width' 속성을 사용할 수 있습니다. 
```
(e.g. logo, background), SVG component 번역해주세요.

sujinleeme commented on manuscript/react-styling/svg.md:
```diff
@@ -63,7 +63,7 @@ const Item = ({ item, onRemoveItem }) => (
 ~~~~~~~
 
-Regardless of the styling approach you are using, you can give your SVG icon in the button a hover effect too. In the basic CSS approach, it would look like the following in the *src/App.css* file:
+여러분이 사용하는 스타일링 기법에 상관없이, SVG 아이콘 버튼에 hover 효과도 줄 수 있습니다. 기본 CSS 기법에서 *src/App.css* 파일에서 아래와 같이 표현할 수 있습니다.
```
번역해주세요. 마우스 오버(hover)

sujinleeme commented on manuscript/react-styling/svg.md:
```diff
@@ -73,12 +73,12 @@ Regardless of the styling approach you are using, you can give your SVG icon in
+create-react-app 프로젝트에서 별도의 설치 없이 SVG를 쉽게 사용할 수 있습니다. 하지만 여러분이 웹팩같은 빌드 도구와 함께 스크래치로 React 프로젝트를 만든다면, 여러분이 직접 관리해야 하므로 다릅니다. SVG는 여러분의 애플리케이션을 좀 더 사용하기 쉽게 만들어 주기 때문에 활용할 수 있을 때마다 사용하길 권장합니다.
 
-### Exercises:
+### 실습하기:
```
실습하기:

뒤에 :를 삭제해주세요.

sujinleeme commented on manuscript/react-styling/svg.md:
```diff
@@ -73,12 +73,12 @@ Regardless of the styling approach you are using, you can give your SVG icon in
-* Add another SVG icon in your application.
+* 마지막 부분의 소스 코드를 확인해보세요. (https://codesandbox.io/s/github/the-road-to-learn-react/hacker-stories/tree/hs/CSS-in-React-SVG)
+* 마지막 부분의 변경사항들을 확인해보세요. (https://github.com/the-road-to-learn-react/hacker-stories/compare/hs/CSS-in-React...hs/CSS-in-React-SVG?expand=1)
+* [SVGs in create-react-app]에 대해 더 알아보세요. (https://create-react-app.dev/docs/adding-images-fonts-and-files)
+* [SVG background patterns in React]에 대해 더 알아보세요. (https://www.robinwieruch.de/react-svg-patterns)
```
제목 번역해주세요.

YeSeulBae committed:
Update svg.md

- react components ->리액트 컴포넌트,
- scalable vector graphic -> 확장 가능한 그래픽으로 변경
- CSS in React 번역
- Flaticon's Freepick. 번역, 뒤에 . 삭제
- e.g. logo, background 번역
- SVG component 번역
- 실습하기 뒤 : 삭제
- SVGs in create-react-app, SVG background patterns in React 번역

YeSeulBae committed:
Delete [v2] SVG 사용

sujinleeme commented on manuscript/react-styling/svg.md:
```diff
@@ -73,12 +73,12 @@ Regardless of the styling approach you are using, you can give your SVG icon in
-* Read more about [SVGs in create-react-app](https://create-react-app.dev/docs/adding-images-fonts-and-files).
-* Read more about [SVG background patterns in React](https://www.robinwieruch.de/react-svg-patterns).
-* Add another SVG icon in your application.
+* 마지막 부분의 소스 코드를 확인해보세요. (https://codesandbox.io/s/github/the-road-to-learn-react/hacker-stories/tree/hs/CSS-in-React-SVG)
+* 마지막 부분의 변경사항들을 확인해보세요. (https://github.com/the-road-to-learn-react/hacker-stories/compare/hs/CSS-in-React...hs/CSS-in-React-SVG?expand=1)
+* [리액트 앱 만들기 속 SVG]에 대해 더 알아보세요. (https://create-react-app.dev/docs/adding-images-fonts-and-files)
+* [리액트 속 SVG 배경 패턴]에 대해 더 알아보세요. (https://www.robinwieruch.de/react-svg-patterns)
```
마크다운 링크가 깨져있습니다.
다른 분들의 링크를 참고해주세요.

sujinleeme replied:
https://github.com/the-road-to-learn-react/the-road-to-react-korean/blob/v2/manuscript/react-modern-advanced/remember-last-searches.md

sujinleeme commented on manuscript/react-styling/svg.md:
```diff
@@ -73,12 +73,12 @@ Regardless of the styling approach you are using, you can give your SVG icon in
 ~~~~~~~
 
-The create-react-app project makes using SVGs straightforward, with no extra configuration needed. This is different if you create a React project from scratch with build tools like Webpack, because you have to take care of it yourself. Anyway, SVGs make your application more approachable, so use them whenever it suits you.
+create-react-app 프로젝트에서 별도의 설치 없이 SVG를 쉽게 사용할 수 있습니다. 하지만 여러분이 웹팩같은 빌드 도구와 함께 스크래치로 React 프로젝트를 만든다면, 여러분이 직접 관리해야 하므로 다릅니다. SVG는 여러분의 애플리케이션을 좀 더 사용하기 쉽게 만들어 주기 때문에 활용할 수 있을 때마다 사용하길 권장합니다.
```
React -> 리액트

sujinleeme commented on manuscript/react-styling/svg.md:
```diff
@@ -1,10 +1,10 @@
+## SVG 사용
 
-To create a modern React application, we'll likely need to use SVGs. Instead of giving every button element text, for instance, we might want to make it lightweight with an icon. In this section, we'll use a scalable vector graphic (SVG) as an icon in one of our React components.
+최신 React 애플리케이션을 만들기 위해, SVG를 사용하는 것이 좋습니다. 모든 버튼 개체 텍스트를 만드는 대신, 예를 들면, 여러분들은 아이콘으로 좀 더 가볍게 만들 수 있습니다. 이번에는 리액트 컴포넌트 중 하나인 확장 가능한 그래픽(SVG)을 사용해 보겠습니다. 
```
React -> 리액트

sujinleeme replied:
확장 가능한 그래픽(SVG, scalable vector graphic) 로 수정해주세요.

sujinleeme commented on manuscript/react-styling/svg.md:
```diff
@@ -1,10 +1,10 @@
+최신 React 애플리케이션을 만들기 위해, SVG를 사용하는 것이 좋습니다. 모든 버튼 개체 텍스트를 만드는 대신, 예를 들면, 여러분들은 아이콘으로 좀 더 가볍게 만들 수 있습니다. 이번에는 리액트 컴포넌트 중 하나인 확장 가능한 그래픽(SVG)을 사용해 보겠습니다. 
 
-This section builds on the "CSS in React" we covered earlier, to give the SVG icon a good look and feel right away. It's acceptable to use a different styling approach, or no styling at all, though the SVG might look off without it.
+이번 장은 SVG 아이콘을 더 멋지고 정확하게 사용할 수 있도록 하는, 이전에 다뤘던 "리액트에서의 CSS"를 기반으로 합니다. SVG는 다른 스타일링 혹은 스타일링이 전혀 없어도 괜찮습니다. 
```
리액트에서의 CSS가 전 챕터와 동일한  제목인가요?

sujinleeme replied:
`SVG는 다른 스타일링 혹은 스타일링이 전혀 없어도 괜찮습니다. ` -> 

잘 못 번역이 된 거 같습니다.

YeSeulBae committed:
Update svg.md

sujinleeme commented on manuscript/react-styling/svg.md:
```diff
@@ -1,8 +1,8 @@
+최신 React 애플리케이션을 만들기 위해, SVG를 사용하는 것이 좋습니다. 모든 버튼 개체 텍스트를 만드는 대신, 예를 들면, 여러분들은 아이콘으로 좀 더 가볍게 만들 수 있습니다. 이번에는 리액트 컴포넌트 중 하나인 확장 가능한 그래픽(SVG, scalable vector graphic)을 사용해 보겠습니다. 
 
-이번 장은 SVG 아이콘을 더 멋지고 정확하게 사용할 수 있도록 하는, 이전에 다뤘던 "리액트에서의 CSS"를 기반으로 합니다. SVG는 다른 스타일링 혹은 스타일링이 전혀 없어도 괜찮습니다. 
+이번 장은 SVG 아이콘을 더 멋지고 정확하게 사용할 수 있도록 하는, 이전에 다뤘던 "리액트에서의 CSS"를 기반으로 합니다. 다른 꾸미기 기법이나, 혹은 전혀 꾸미지 않아도 되며, SVG는 꾸미기 없이도 괜찮습니다. 
```
꾸미기 기법이라는 표현은 저는 낯선데 스타일링이 낫지 않을까요? 이렇게 바꾸신 이유가 있나요?

yeseulo commented on manuscript/react-styling/svg.md:
```diff
@@ -23,7 +23,7 @@ This icon as SVG is taken from [Flaticon's Freepick](https://www.flaticon.com/au
 ~~~~~~~
 
-Because we're using create-react-app again , we can  import SVGs (similar to CSS) as React components right away. In *src/App.js*, use the following syntax for importing the SVG:
+create-react-app을 다시 쓰기 때문에, 리액트 컴포넌트(CSS와 유사한)로 SVG를 바로 가져올 수 있습니다. *src/App.js*에서 SVG를 가져오기 위해 아래 문법을 사용합니다.
```
국문에서 괄호 위치가 어색해서 위치를 문장 끝으로 바꾸거나 풀어써도 괜찮을 듯 합니다.
```diff
- 리액트 컴포넌트(CSS와 유사한)로 SVG를 바로 가져올 수 있습니다.
+ 바로 리액트 컴포넌트로 SVG를 가져올 수 있습니다(CSS와 비슷하게).
+ 바로 리액트 컴포넌트로 CSS와 비슷하게 SVG를 가져올 수 있습니다.
```

yeseulo commented on manuscript/react-styling/svg.md:
```diff
@@ -36,7 +36,7 @@ import { ReactComponent as Check } from './check.svg';
 ~~~~~~~
 
-We are importing an SVG, and this works for many different uses for SVGs (e.g. logo, background). Instead of a button text, pass the SVG component as a `height` and `width` attribute:
+SVG를 가져오게 되면, (예. 로고, 배경) 다양한 SVG를 활용할 수 있습니다. 버튼 텍스트 대신, SVG 컴포넌트로 'height'와 'width' 속성을 사용할 수 있습니다. 
```
마찬가지로 괄호 위치를 옮기면 좋겠습니다.
```diff
- SVG를 가져오게 되면, (예. 로고, 배경) 다양한 SVG를 활용할 수 있습니다.
+ SVG를 가져와서(예. 로고, 배경) 다양한 용도로 활용할 수 있습니다.
```
뒷 문장도 의미에 맞게 수정해봤습니다.
```diff
- 버튼 텍스트 대신, SVG 컴포넌트로 'height'와 'width' 속성을 사용할 수 있습니다.
+ 버튼에 텍스트 대신 'height'와 'width' 속성이 있는 SVG 컴포넌트를 사용할 수 있습니다.
```

yeseulo commented on manuscript/react-styling/svg.md:
```diff
@@ -1,10 +1,10 @@
+이번 장은 SVG 아이콘을 더 멋지고 정확하게 사용할 수 있도록 하는, 이전에 다뤘던 "리액트에서의 CSS"를 기반으로 합니다. 다른 꾸미기 기법이나, 혹은 전혀 꾸미지 않아도 되며, SVG는 꾸미기 없이도 괜찮습니다. 
 
-This icon as SVG is taken from [Flaticon's Freepick](https://www.flaticon.com/authors/freepik). Many of the SVGs on this website are free to use, though they require you to mention the author. You can download the icon from [here](https://www.flaticon.com/free-icon/check_109748) as SVG and put it in your project as *src/check.svg*. Downloading the file is the recommended way, however, for the sak ...
+이 SVG 아이콘은 [플랫아이콘 무료 사이트](https://www.flaticon.com/authors/freepik)에 있습니다. 이 사이트의 많은 SVG는 원작자를 남기면 무료로 사용할 수 있습니다. (https://www.flaticon.com/free-icon/check_109748) 여기서 SVG로 아이콘을 다운로드 할 수 있으며 프로젝트 *src/check.svg* 파일에 적용하면 됩니다. 이 파일을 다운로드 받는 것이 좋으나, 더 자세히 알기 위해 SVG 정의를 살펴보면 아래와 같습니다.
```
```diff
- SVG로 아이콘을 다운로드 할 수 있으며
+ SVG로 아이콘을 다운로드할 수 있으며
```
```diff
- 이 파일을 다운로드 받는 것이 좋으나,
+ 이 파일을 다운로드하는 것이 좋으나,
```

yeseulo commented on manuscript/react-styling/svg.md:
```diff
@@ -73,12 +73,12 @@ Regardless of the styling approach you are using, you can give your SVG icon in
 ~~~~~~~
 
-The create-react-app project makes using SVGs straightforward, with no extra configuration needed. This is different if you create a React project from scratch with build tools like Webpack, because you have to take care of it yourself. Anyway, SVGs make your application more approachable, so use them whenever it suits you.
+create-react-app 프로젝트에서 별도의 설치 없이 SVG를 쉽게 사용할 수 있습니다. 하지만 여러분이 웹팩같은 빌드 도구와 함께 스크래치로 리액트 프로젝트를 만든다면, 여러분이 직접 관리해야 하므로 다릅니다. SVG는 여러분의 애플리케이션을 좀 더 사용하기 쉽게 만들어 주기 때문에 활용할 수 있을 때마다 사용하길 권장합니다.
```
```diff
- 웹팩같은
+ 웹팩 같은
```

YeSeulBae committed:
Update svg.md

YeSeulBae committed:
Update svg.md

sujinleeme commented on manuscript/react-styling/svg.md:
```diff
@@ -1,10 +1,10 @@
+이번 장은 SVG 아이콘을 더 멋지고 정확하게 사용할 수 있도록 하는, 이전에 다뤘던 "리액트에서의 CSS"를 기반으로 합니다. 다른 스타일링이나, 혹은 전혀 꾸미지 않아도 되며, SVG는 꾸미기 없이도 괜찮습니다. 
 
-This icon as SVG is taken from [Flaticon's Freepick](https://www.flaticon.com/authors/freepik). Many of the SVGs on this website are free to use, though they require you to mention the author. You can download the icon from [here](https://www.flaticon.com/free-icon/check_109748) as SVG and put it in your project as *src/check.svg*. Downloading the file is the recommended way, however, for the sak ...
+이 SVG 아이콘은 [플랫아이콘 무료 사이트](https://www.flaticon.com/authors/freepik)에 있습니다. 이 사이트의 많은 SVG는 원작자를 남기면 무료로 사용할 수 있습니다. (https://www.flaticon.com/free-icon/check_109748) 여기서 SVG로 아이콘을 다운로드할 수 있으며 프로젝트 *src/check.svg* 파일에 적용하면 됩니다. 이 파일을 다운로드하는 것이 좋으나, 더 자세히 알기 위해 SVG 정의를 살펴보면 아래와 같습니다.
```
마크다운 링크가 깨져있어요.

`[여기서](https://www.flaticon.com/free-icon/check_109748) `

YeSeulBae committed:
Update svg.md

마크다운 변경

sujinleeme commented on manuscript/react-styling/svg.md:
```diff
@@ -73,12 +73,12 @@ Regardless of the styling approach you are using, you can give your SVG icon in
-* Read more about [SVGs in create-react-app](https://create-react-app.dev/docs/adding-images-fonts-and-files).
-* Read more about [SVG background patterns in React](https://www.robinwieruch.de/react-svg-patterns).
-* Add another SVG icon in your application.
+* 마지막 장의 소스 코드를 확인합니다. (https://codesandbox.io/s/github/the-road-to-learn-react/hacker-stories/tree/hs/CSS-in-React-SVG)
+* 마지막 장에서 변경된 코드를 확인합니다. (https://github.com/the-road-to-learn-react/hacker-stories/compare/hs/CSS-in-React...hs/CSS-in-React-SVG?expand=1)
+* [리액트 앱 만들기 속 SVG]에 대해 더 알아보세요. (https://create-react-app.dev/docs/adding-images-fonts-and-files)
+* [리액트 속 SVG 배경 패턴]에 대해 더 알아보세요. (https://www.robinwieruch.de/react-svg-patterns)
+* 다른 SVG 아이콘을 애플리케이션에 적용해보세요.
```
마크다운 링크가 깨져있습니다. 수정해주세요.

sujinleeme commented on manuscript/react-styling/svg.md:
```diff
@@ -73,12 +73,12 @@ Regardless of the styling approach you are using, you can give your SVG icon in
 ~~~~~~~
 
-The create-react-app project makes using SVGs straightforward, with no extra configuration needed. This is different if you create a React project from scratch with build tools like Webpack, because you have to take care of it yourself. Anyway, SVGs make your application more approachable, so use them whenever it suits you.
+create-react-app 프로젝트에서 별도의 설치 없이 SVG를 쉽게 사용할 수 있습니다. 하지만 여러분이 웹팩 같은 빌드 도구와 함께 스크래치로 리액트 프로젝트를 만든다면, 여러분이 직접 관리해야 하므로 다릅니다. SVG는 여러분의 애플리케이션을 좀 더 사용하기 쉽게 만들어 주기 때문에 활용할 수 있을 때마다 사용하길 권장합니다.
```
-> 따라서 SVG이 편리하므로 적극적으로 사용하길 권장합니다. 가 어떨까요.

yeseulo replied:
별도의 설치 없이 -> '설치'보다는 '구성' 이나 '설정' 이 어떨까요

sujinleeme commented on manuscript/react-styling/svg.md:
```diff
@@ -23,7 +23,7 @@ This icon as SVG is taken from [Flaticon's Freepick](https://www.flaticon.com/au
 ~~~~~~~
 
-Because we're using create-react-app again , we can  import SVGs (similar to CSS) as React components right away. In *src/App.js*, use the following syntax for importing the SVG:
+create-react-app을 다시 쓰기 때문에, 바로 리액트 컴포넌트로 CSS와 비슷하게 SVG를 가져올 수 있습니다. *src/App.js*에서 SVG를 가져오기 위해 아래 문법을 사용합니다.
```
불필요한 (,) 제거해주시고 맞춤법 검사기 확인해주세요.

sujinleeme commented on manuscript/react-styling/svg.md:
```diff
@@ -1,10 +1,10 @@
+## SVG 사용
 
-To create a modern React application, we'll likely need to use SVGs. Instead of giving every button element text, for instance, we might want to make it lightweight with an icon. In this section, we'll use a scalable vector graphic (SVG) as an icon in one of our React components.
+최신 React 애플리케이션을 만들기 위해, SVG를 사용하는 것이 좋습니다. 모든 버튼 개체 텍스트를 만드는 대신, 예를 들면, 여러분들은 아이콘으로 좀 더 가볍게 만들 수 있습니다. 이번에는 리액트 컴포넌트 중 하나인 확장 가능한 그래픽(SVG, scalable vector graphic)을 사용해 보겠습니다. 
```
React -> 리액트로 수정해주시고 불필요한 컴마를 모두 삭제해주세요. 맞춤법 검사기 확인해주세요.

sujinleeme commented on manuscript/react-styling/svg.md:
```diff
@@ -1,10 +1,10 @@
+이번 장은 SVG 아이콘을 더 멋지고 정확하게 사용할 수 있도록 하는, 이전에 다뤘던 "리액트에서의 CSS"를 기반으로 합니다. 다른 스타일링이나, 혹은 전혀 꾸미지 않아도 되며, SVG는 꾸미기 없이도 괜찮습니다. 
 
-This icon as SVG is taken from [Flaticon's Freepick](https://www.flaticon.com/authors/freepik). Many of the SVGs on this website are free to use, though they require you to mention the author. You can download the icon from [here](https://www.flaticon.com/free-icon/check_109748) as SVG and put it in your project as *src/check.svg*. Downloading the file is the recommended way, however, for the sak ...
+이 SVG 아이콘은 [플랫아이콘 무료 사이트](https://www.flaticon.com/authors/freepik)에 있습니다. 이 사이트의 많은 SVG는 원작자를 남기면 무료로 사용할 수 있습니다. [여기서](https://www.flaticon.com/free-icon/check_109748) SVG로 아이콘을 다운로드할 수 있으며 프로젝트 *src/check.svg* 파일에 적용하면 됩니다. 이 파일을 다운로드하는 것이 좋으나, 더 자세히 알기 위해 SVG 정의를 살펴보면 아래와 같습니다.
```
여기서 -> 이 곳에서

sujinleeme commented on manuscript/react-styling/svg.md:
```diff
@@ -36,7 +36,7 @@ import { ReactComponent as Check } from './check.svg';
 ~~~~~~~
 
-We are importing an SVG, and this works for many different uses for SVGs (e.g. logo, background). Instead of a button text, pass the SVG component as a `height` and `width` attribute:
+SVG를 가져와서(예. 로고, 배경) 다양한 용도로 활용할 수 있습니다. 버튼에 텍스트 대신 'height'와 'width' 속성이 있는 SVG 컴포넌트를 사용할 수 있습니다.
```
SVG를 가져와서 로고, 배경 등 다양한 용도로 활용할 수 있습니다.  -> 으로 수정해주세요.

YeSeulBae committed:
Update svg.md

yeseulo commented on manuscript/react-styling/svg.md:
```diff
@@ -73,12 +73,12 @@ Regardless of the styling approach you are using, you can give your SVG icon in
 ~~~~~~~
 
-The create-react-app project makes using SVGs straightforward, with no extra configuration needed. This is different if you create a React project from scratch with build tools like Webpack, because you have to take care of it yourself. Anyway, SVGs make your application more approachable, so use them whenever it suits you.
+create-react-app 프로젝트에서 별도의 설정 없이 SVG를 쉽게 사용할 수 있습니다. 하지만 여러분이 웹팩 같은 빌드 도구와 함께 스크래치로 리액트 프로젝트를 만든다면 여러분이 직접 관리해야 하므로 다릅니다. SVG는 여러분의 애플리케이션을 좀 더 사용하기 쉽게 만들어 주기 때문에 적극적으로 사용하길 권장합니다.
```
```diff
- 하지만 여러분이 웹팩 같은 빌드 도구와 함께 스크래치로 리액트 프로젝트를 만든다면
+ 하지만 여러분이 웹팩 같은 빌드 도구와 함께 리액트 프로젝트를 처음부터 만든다면
```

YeSeulBae committed:
Update svg.md

sujinleeme commented on manuscript/react-styling/svg.md:
```diff
@@ -1,10 +1,10 @@
+이번 장은 SVG 아이콘을 더 멋지고 정확하게 사용할 수 있도록 하는 이전에 다뤘던 "리액트에서의 CSS"를 기반으로 합니다. 다른 스타일링이나 혹은 전혀 꾸미지 않아도 되며 SVG는 꾸미기 없이도 괜찮습니다. 
 
-This icon as SVG is taken from [Flaticon's Freepick](https://www.flaticon.com/authors/freepik). Many of the SVGs on this website are free to use, though they require you to mention the author. You can download the icon from [here](https://www.flaticon.com/free-icon/check_109748) as SVG and put it in your project as *src/check.svg*. Downloading the file is the recommended way, however, for the sak ...
+이 SVG 아이콘은 [플랫아이콘 무료 사이트](https://www.flaticon.com/authors/freepik)에 있습니다. 이 사이트의 많은 SVG는 원작자를 남기면 무료로 사용할 수 있습니다. [이 곳에서](https://www.flaticon.com/free-icon/check_109748) SVG로 아이콘을 다운로드할 수 있으며 프로젝트 *src/check.svg* 파일에 적용하면 됩니다. 이 파일을 다운로드하는 것이 좋으나 더 자세히 알기 위해 SVG 정의를 살펴보면 아래와 같습니다.
```
```
[이 곳에서](https://www.flaticon.com/free-icon/check_109748)
==> [이 곳](https://www.flaticon.com/free-icon/check_109748)에서
```

sujinleeme replied:
이 사이트의 많은 SVG는 원작자를 남기면 무료로 사용할 수 있습니다. 
--> 무슨 말인지 모르겠습니다. 수정부탁드려요

sujinleeme commented on manuscript/react-styling/svg.md:
```diff
@@ -1,10 +1,10 @@
+최신 리액트 애플리케이션을 만들기 위해 SVG를 사용하는 것이 좋습니다. 모든 버튼 개체 텍스트를 만드는 대신 예를 들면 여러분들은 아이콘으로 좀 더 가볍게 만들 수 있습니다. 이번에는 리액트 컴포넌트 중 하나인 확장 가능한 그래픽(SVG, scalable vector graphic)을 사용해 보겠습니다. 
 
-This section builds on the "CSS in React" we covered earlier, to give the SVG icon a good look and feel right away. It's acceptable to use a different styling approach, or no styling at all, though the SVG might look off without it.
+이번 장은 SVG 아이콘을 더 멋지고 정확하게 사용할 수 있도록 하는 이전에 다뤘던 "리액트에서의 CSS"를 기반으로 합니다. 다른 스타일링이나 혹은 전혀 꾸미지 않아도 되며 SVG는 꾸미기 없이도 괜찮습니다. 
```
"리액트에서의 CSS" 이 책의 제목과 일치하게 수정해주세요.
꾸미기 삭제--> 스타일링으로 고쳐주세요. 문장이 이상합니다.

sujinleeme committed:
update contents
